src/serverlessFunction/functions.ts

```diff
@@ -63,31 +63,38 @@ export class Functions {
                     });
 
                     // start new build
-                    await this.buildProcess(context, view);
+                    await this.buildProcess(context);
                 }
             });
         } else {
-            await this.buildProcess(context, view);
+            await this.buildProcess(context);
         }
     }
 
-    private async buildProcess(context: FunctionObject, view: FunctionView) {
+    private async buildProcess(context: FunctionObject) {
         const isOpenShiftCluster = await Oc.Instance.isOpenShiftCluster();
         const buildImage = await this.getImage(context.folderURI);
         const terminalKey = `build-${context.folderURI.fsPath}`;
 
+        const specificPlatform = await window.showQuickPick(['Yes', 'No'], {
+            placeHolder: 'Do you want to build it for \'linux/amd64\' platform?',
```

Comment:
It's not very clear what select "No" does in this case, especially when you are on a Linux/amd64 machine. I think this message should be improved to explain what happens if you pick "No"

Reply:
For better user experience added the context menu with build config. 
![image](https://github.com/redhat-developer/vscode-openshift-tools/assets/93245779/0d9e8369-d3b3-4442-8f8e-3e0a458264d7)